runner_manager/models/runner_group.py

```diff
@@ -129,9 +130,12 @@ def create_runner(self, github: GitHub) -> Runner | None:
             )
             runner.save()
             runner.generate_jit_config(github)
-
+            if self.queued > 0:
+                self.queued -= 1
             return self.backend.create(runner)
-        return None
+        else:
+            self.queued += 1
+            return None
```

Comment:
Can it be negative? Or else is =0?
And don't need to return None, just return

Reply:
else is we couldn't created a runner because we reached the maximum allowed so we're adding it up.

There's a part of the code missing to review this part accordingly due to the diff size:
```python
if count < self.max and self.id
```